src/data/faq_de.json

```diff
@@ -158,11 +158,23 @@
                         "active": true,
                         "textblock": [
+                            "<b>Aktuell</b><br/>Bitte beachten Sie das sich das Verhalten der Zertifikatsgültigkeitsprüfung mit Version 2.8 geändert hat. Weitere Informationen zu dieser Änderung finden Sie in dem FAQ-Eintrag: <a href='#dcc_validity_check_change' target='_blank' rel='noopener noreferrer'>Warum zeigt die Corona-Warn-App auf Version 2.8 an, dass mein Zertifikat nicht prüfbar ist, obwohl die vorherige Version mein Zertifikat als gültig anzeigte?</a><hr>",
                             "Sie können ab der Version 2.6 vor einer Reise prüfen, ob Ihre Zertifikate (Test-, Genesenen-, und/oder Impfzertifikat) in einem ausgewählten Land zum Zeitpunkt Ihrer Reise gültig sind. Die Corona-Warn-App berücksichtigt dafür die geltenden Einreiseregeln des ausgewählten Reiselandes und gleicht sie mit verschiedenen Parametern des Zertifikats, wie Datum und Art des Tests, Testzentrum oder Datum einer Impfung, ab.",
                             "Jedes europäische Land, das digitale COVID-Zertifikate der EU unterstützt, hat die Möglichkeit Regeln hochzuladen, die die Corona-Warn-App zur Überprüfung abgleichen kann. Zum Start der neuen Funktion haben Irland, Litauen, Luxemburg, die Niederlande, Slowenien und Spanien, zusammen mit Deutschland, Regeln hinterlegt, weitere Länder folgen in den kommenden Wochen.",
                             "Zeigt die App nach Prüfung an, dass keine Einreiseregeln vorhanden sind, können Sie sich unter <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>https://reopen.europa.eu/de</a> über die Regeln informieren."
                         ]
                     },
+                    {
+                        "title": "Warum zeigt die Corona-Warn-App auf Version 2.8 an, dass mein Zertifikat nicht prüfbar ist, obwohl die vorherige Version mein Zertifikat als gültig anzeigte?",
+                        "anchor": "ddc_no_rules",
+                        "active": true,
+                        "textblock": [
+                            "Mit der Version 2.8 hat sich das Verhalten der Zertifikatsgültigkeitsprüfung geändert. Einreiseregeln für Reisende wurden von allen EU-Mitgliedsländern aufgestellt, aber bis heute haben nur wenige Länder Einreiseregeln in einem maschinenlesbaren Format bereitgestellt, die die Corona-Warn-App nutzen könnte. Vor Version 2.8, wenn für ein bestimmtes Land keine Einreiseregeln in einem maschinenlesbaren Format zur Verfügung standen, zeigte die Corona-Warn-App nach einer Überprüfung 'Gültiges Zertifikat' an und erklärte im folgenden Text, dass für dieses Land keine Einreiseregeln zur Verfügung gestellt wurden/überprüft werden konnten. 'Gültiges Zertifikat' bezog sich in diesem Fall auf die ’Technische Gültigkeit' (mehr Informationen darüber, was die ‚Technische Gültigkeit' ist, finden Sie in <a href='#eu_dcc_validity' target='_blank' rel='noopener noreferrer'>diesem</a> FAQ-Eintrag).",

```

Comment:
```suggestion
                            "Mit der Version 2.8 hat sich das Verhalten der Zertifikatsgültigkeitsprüfung geändert. Die Regeln zur Akzeptanz der digitalen Zertifikate im Zielland wurden inzwischen von allen EU-Mitgliedsländern definiert. Leider haben noch nicht alle Länder diese Regeln für eine maschinelle Verarbeitung auf dem EU-Gateway bereitgestellt. Vor Version 2.8 zeigte die Corona-Warn-App mit dem grünen Checkmark ein Zertifikat als "gültig" an, selbst wenn keine Akzeptanz-Regeln geprüft werden konnten. Das Ergebnis "gültig" bedeutete dabei, dass das Zertifikat noch nicht abgelaufen war ("technische Gültigkeit" - mehr Informationen dazu finden sie in <a href='#eu_dcc_validity' target='_blank' rel='noopener noreferrer'>diesem</a> FAQ-Eintrag).",

```

Reply:
The change to "finden sie" is grammatically wrong. It should stay as "finden Sie".